### PR TITLE
Add Order and Address endpoints

### PR DESCRIPTION
**Pull request by carlosfelipeg**

### CODE REVIEW

File: lib/drivin/resources/order.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Drivin
+  module Resources
+    module Order
+      extend Resource
+
+      def self.create_by_schema_code(schema_code, params)
+        post("api/external/v2/orders?schema_code=#{schema_code}", params)
+      end
```
Comment (alejandrocen) [inline lib/drivin/resources/order.rb]: Move this method to a specific namespace:

```ruby
Drivin::Resources::Order::SchemaCode.create("schema_code")

# Alias usage:
Drivin::Order::SchemaCode.create("schema_code")
```

File: lib/drivin/resources/order.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+      def self.create_by_scenario_token(scenario_token, params)
+        post("api/external/v2/orders?token=#{scenario_token}", params)
+      end
```
Comment (alejandrocen) [inline lib/drivin/resources/order.rb]: The same fix:


```ruby

Drivin::Order::ScenarioToken.create("token")


```

File: lib/drivin/resources/order.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+      def self.get_by_token(token)
+        get("api/external/v2/orders?token=#{token}")
+      end
```
Comment (alejandrocen) [inline lib/drivin/resources/order.rb]: The same fix:


```ruby

Drivin::Order::ScenarioToken.find("token")


```

File: lib/drivin/resources/order.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+      def self.delete(order_code)
+        delete("api/external/v2/orders/#{order_code}")
+      end
```
Comment (alejandrocen) [inline lib/drivin/resources/order.rb]:
```suggestion
      def self.destroy(order_code)
        delete("api/external/v2/orders/#{order_code}")
      end
```

File: lib/drivin/resources/order.rb [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+      def self.delete_by_scenario_token(order_code, scenario_token)
+        delete("api/external/v2/orders/#{order_code}?token=#{scenario_token}")
+      end
```
Comment (alejandrocen) [inline lib/drivin/resources/order.rb]: The same fix:


```ruby

Drivin::Order::ScenarioToken.destroy("token")


```

File: lib/drivin/resources/user.rb [hunk 6]
```diff
@@ -8,6 +8,14 @@ module User
       def self.create(params)
         post('api/external/v2/users', params)
       end
+
+      def self.get
+        get('api/external/v2/users')
+      end
```
Comment (alejandrocen) [inline lib/drivin/resources/user.rb]:
```suggestion
      def self.all(params = EMPTY_PARAMS.dup)
        get('api/external/v2/users', params)
      end
```


```ruby
# Usage

Drivin::User.all
Drivin::User.all(email: "email@email.com")
Drivin::User.all(role_name: "admin")
```

File: lib/drivin/resources/user.rb [hunk 7]
```diff
@@ -8,6 +8,14 @@ module User
...
+
+      def self.get_by_role(role_name)
+        get("api/external/v2/users?role_name=#{role_name}")
+      end
```
Comment (alejandrocen) [inline lib/drivin/resources/user.rb]: Remove this method, use .all method with query strings

**Review by alejandrocen — CHANGES_REQUESTED** [review]

File: lib/drivin/resources/user.rb (hunk 6 above)
Comment (carlosfelipeg) [reply lib/drivin/resources/user.rb]: https://apidocs.driv.in/#8877ab80-d414-4a7a-a441-44bbd5d793d8
No se si podamos filtrar tanto, creo que solo es por rol

**Review by alejandrocen — APPROVED** [review]

**Pull request merged**